### DISCUSSION TITLE
Getting a documents probability vector when using QL retrieval

### DISCUSSION THREAD

**Discussion by noamaon** [castorini/pyserini#2036] [Q&A]
Is there a way to access the term distributions created when using a Bayesian smoothed language model on indexed documents? If I understand correctly, these distributions should be created when running search with the --qld argument.

**Comment by lintool** [comment]
Hi @noamaon what exactly do you mean you want the term distributions? Like the term vector? That can be accessed as follows:
https://github.com/castorini/pyserini/blob/master/docs/usage-indexreader.md#how-do-i-access-and-manipulate-term-vectors
